### PR TITLE
feat(core): useMaskedErrors plugin

### PR DESCRIPTION
**Pull request by n1ru4l**
Mask unexpected errors that occur during execution

### CODE REVIEW

File: packages/core/src/index.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-export * from './plugins/use-payload-formatter';
+export * from '@envelop/types';
+export * from './create';
+export * from './plugins/use-logger';
+export * from './plugins/use-timing';
+export * from './plugins/use-schema';
+export * from './plugins/use-error-handler';
+export * from './plugins/use-extend-context';
+export * from './plugins/use-payload-formatter';
+export * from './plugins/use-masked-errors';
```
Comment (PabloSzx) [inline packages/core/src/index.ts]: crlf commits?

Comment (n1ru4l) [reply packages/core/src/index.ts]: ahhh f windows :(

Comment (PabloSzx) [reply packages/core/src/index.ts]: if husky is added with pretty-quick, this issue won't ever happen 

@dotansimha can I create a PR?

Comment (n1ru4l) [reply packages/core/src/index.ts]: fixed 🕺

**Review by dotansimha — APPROVED** [review]

**Pull request merged**